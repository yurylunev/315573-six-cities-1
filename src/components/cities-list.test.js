import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list.jsx";

const cities = [
  {id: 0, cityName: `Citie 1`},
  {id: 1, cityName: `Citie 2`},
  {id: 2, cityName: `Citie 3`},
  {id: 3, cityName: `Citie 4`}
];

it(`CitiesList correctly renders`, () => {
  const tree = renderer
    .create(<CitiesList cities={cities} clickHandler={jest.fn()} currentId={0}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
