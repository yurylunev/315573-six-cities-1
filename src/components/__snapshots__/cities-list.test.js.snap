// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList correctly renders 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
      onClick={[Function]}
    >
      <a
        className="locations__item-link tabs__item tabs__item--active"
      >
        <span>
          Citie 1
        </span>
      </a>
    </li>
    <li
      className="locations__item"
      onClick={[Function]}
    >
      <a
        className="locations__item-link tabs__item"
      >
        <span>
          Citie 2
        </span>
      </a>
    </li>
    <li
      className="locations__item"
      onClick={[Function]}
    >
      <a
        className="locations__item-link tabs__item"
      >
        <span>
          Citie 3
        </span>
      </a>
    </li>
    <li
      className="locations__item"
      onClick={[Function]}
    >
      <a
        className="locations__item-link tabs__item"
      >
        <span>
          Citie 4
        </span>
      </a>
    </li>
  </ul>
</section>
`;
